src/Resources/ClientLink.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Mollie\Api\Resources;
+
+use Mollie\Api\Types\ApprovalPrompt;
+
+class ClientLink extends BaseResource
+{
+    /**
+     * @var string
+     */
+    public $resource;
+
+    /**
+     * Id of the client link.
+     *
+     * @example csr_vZCnNQsV2UtfXxYifWKWH
+     * @var string
+     */
+    public $id;
+
+    /**
+     * An object with several URL objects relevant to the client link. Every URL object will contain an href and a type field.
+     *
+     * @var \stdClass
+     */
+    public $_links;
+
+    /**
+     * Get the redirect URL where the customer can complete the payment.
+     *
+     * @return string|null
+     */
+    public function getRedirectUrl(string $client_id, string $state, string $approval_prompt, array $scopes = [])
+    {
+        if (! in_array($approval_prompt, [ApprovalPrompt::AUTO, ApprovalPrompt::FORCE])) {
+            throw new \Exception('Invalid approval_prompt. Please use "auto" or "force".');
+        }
+
+        $query = http_build_query([
+            'client_id' => $client_id,
+            'state' => $state,
+            'approval_prompt' => $approval_prompt,
+            'scopes' => implode('+', $scopes),
```

Comment:
There is a typo in our documentation, it should be `scope` instead of `scopes`. 
I'll also fix the docs.

Also, I had to change the character to implode, in the example in the documentation we have the encoded URL.
`http_build_query` encodes space as `+`. 

```suggestion
            'scope' => implode(' ', $scopes),
```